src/core/validations/message.rs

```diff
@@ -208,6 +208,9 @@ pub fn validate_message(
             validate_frame_action_body(&frame_action_body)?;
         }
         Some(proto::message_data::Body::LendStorageBody(lend_storage_body)) => {
+            if message_data.timestamp < block_timestamp.decr_by(60 * 60).to_u64() as u32 {
```

Comment:
Let's add a comment about why we need this, especially just for StorageLend. I also wonder if this should be more prominent, in the engine vs hidden away within validations.